ui/functions.html

```diff
@@ -133,6 +133,8 @@
                 $('#no-functions').show()
                 $('#create-link').href = "/create" + window.location.search
```

Comment:
Looking forward to replacing this url param auth business with a cookie or something once we have a proper user/auth system